Client/multiplayer_sa/CMultiplayerSA_Direct3D.cpp

```diff
@@ -151,6 +141,6 @@ void _declspec(naked) HOOK_PostCreateDevice()
 //////////////////////////////////////////////////////////////////////////////////////////
 void CMultiplayerSA::InitHooks_Direct3D()
 {
-    EZHookInstall(PreCreateDevice);
```

Comment:
What changes this?

Reply:
Previously, EZHookInstall was saving bytes to restore the hooks if a variable with the name "RESTORE_Bytes_HookName" existed for US/EU hooks. However, this functionality was only used for the PreCreateDevice hook. Therefore, I moved it into its own macro so we don't have unnecessary code generated for all hooks.